src/test/java/seedu/address/model/task/StatusTest.java

```diff
@@ -9,18 +8,12 @@
 
     @Test
     public void isValidStatus() {
-        // blank email
-        assertFalse(Status.isValidStatus("")); // empty string
-        assertFalse(Status.isValidStatus(" ")); // spaces only
-
-        // valid email
-        assertTrue(Status.isValidStatus("PeterJack_1190@WEB.Mail.com"));
-        assertTrue(Status.isValidStatus("a@b"));  // minimal
-        assertTrue(Status.isValidStatus("test@localhost"));   // alphabets only
-        assertTrue(Status.isValidStatus("123@145"));  // numeric local part and domain name
-        assertTrue(Status.isValidStatus("a1@sg50.org"));  // mixture of alphanumeric and dot characters
-        assertTrue(Status.isValidStatus("_user_@_do_main_.com_"));    // underscores
-        assertTrue(Status.isValidStatus("peter_jack@a_very_long_domain_AVLD.com"));   // long domain name
-        assertTrue(Status.isValidStatus("if.you.dream.it_you.can.do.it@youth_email.com"));    // long local part
+        // valid status
+        assertTrue(Status.isValidStatus("Incomplete"));
+        assertTrue(Status.isValidStatus("done"));
+        assertTrue(Status.isValidStatus("completed"));
+        assertTrue(Status.isValidStatus("finished"));
+        assertTrue(Status.isValidStatus("1/2 done")); // alphanumeric
+        assertTrue(Status.isValidStatus("pending"));
```

Comment:
Hmm thinking of just leaving it first if we are going to restrict the parameters of status in subsequent milestones. Will note this down!